pkg/condition/condition.go

```diff
@@ -0,0 +1,25 @@
+package condition
+
+import (
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+)
+
+func IsConditionTrue(conditions []toolchainv1alpha1.Condition, conditionType toolchainv1alpha1.ConditionType) bool {
+	for _, cond := range(conditions) {
```

Comment:
Oh I just learnt that `range(conditions)` is also fine. I generally seen this thing as `range conditions` :-)

Reply:
yes, me too. And I believe it's more readable without parenthesis

Reply:
agree with you @xcoulon 
with the parenthesis, it looks like Java :-)